Model/IO/Export/VTK_exporter/VTK_Exporter.cpp

```diff
@@ -0,0 +1,46 @@
+#include "VTK_Exporter.h"
+
+void VTK_Exporter::Export(const string & absolute_file_path, const CMR & mesh_data)
```

Comment:
This function is a little to long. It will be clearer, if several sections will be handled by separate functions.